internal/cmd/secrets-manager/instance/create/create.go

```diff
@@ -109,3 +129,17 @@ func buildRequest(ctx context.Context, model *inputModel, apiClient *secretsmana
 
 	return req
 }
+
+func buildUpdateACLsRequest(ctx context.Context, model *inputModel, instanceId string, apiClient *secretsmanager.APIClient) secretsmanager.ApiUpdateACLsRequest {
+	req := apiClient.UpdateACLs(ctx, model.ProjectId, instanceId)
+
+	cidrs := []secretsmanager.AclUpdate{}
```

Comment:
*Small nitpick:* In cases where we already know the size of the slice, we can use `make([]secretsmanager.AclUpdate{}, LENGTH)` to define the length instead of allocating more memory each time we want to append an item

In this case we would then do `cidrs[i] = secretsmanager.AclUpdate{Cidr: utils.Ptr(acl)}` to set each element